api/v1/utils/scopes.js

```diff
@@ -0,0 +1,4 @@
+// TODO: Define all the token scopes that we'll be using 
+var ALL_SCOPES = ['AUTH', 'OTHER'];
+
+module.exports.AUTH = 'AUTH'; // Default scope
```

Comment:
Why is this the default scope? Anyhow, as I said above somewhere, I don't think we should have a default scope.